README.md

```diff
@@ -31,15 +31,16 @@ For a more detailed description, and dependencies, visit [How to Build Transmiss
 
 ### Building a Transmission release from the command line
 
-    $ tar xf transmission-3.00.tar.xz
-    $ cd transmission-3.00
-    $ mkdir build
-    $ cd build
-    # Use -DCMAKE_BUILD_TYPE=RelWithDebInfo to build optimized binary with debug information. (preferred)
-    # Use -DCMAKE_BUILD_TYPE=Release to build full optimized binary.
-    $ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
-    $ make
-    $ sudo make install
+```bash
+$ tar xf transmission-4.0.4.tar.xz
+$ cd transmission-4.0.4
+# Use -DCMAKE_BUILD_TYPE=RelWithDebInfo to build optimized binary with debug information. (preferred)
+# Use -DCMAKE_BUILD_TYPE=Release to build full optimized binary.
+$ cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
+$ cd build
+$ cmake --build .
+$ sudo cmake --install .
```

Comment:
You're building twice.

Can you limit it to max two instructions?
```
cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo
cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo --install
```

Reply:
I prefer using Ninja; but for the general-purpose-build-instructions-for-everyone `README.md`, it's probably better to omit any steps that aren't absolutely required.

Reply:
commenting again for clarification, I think I missed an important part: @Coeur will ninja be a build requirement due to Swift?

I'm OK with the idea, just making sure I understand correctly

Reply:
For the macOS app builds, `-G Ninja` OR `-G Xcode` will be required for Swift (`make` won't work). But ninja won't be strictly required to build: the Xcode option makes it buildable from command line without NInja.

Reply:
If the current build system continues to work, shouldn't the proposed changes be added to the readme rather than replacing?

Reply:
I agree that the `README.md` build instructions should be as general as possible. Let's keep it as-is until the Swift PRs come.